app/components-react/windows/settings/Video.tsx

```diff
@@ -390,6 +390,10 @@ class VideoSettingsModule {
   handleAuth() {
     Services.WindowsService.actions.closeChildWindow();
     this.userService.actions.showLogin();
+    Services.OnboardingService.onboardingCompleted.subscribe(() => {
+      Services.DualOutputService.actions.setdualOutputMode();
+      Services.SettingsService.actions.showSettings('Video');
+    });
```

Comment:
i think you'll probably want to unsubscribe from this subscription while in the callback

Reply:
Done